internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/hadoop/input/kafka/TestKafkaInputFormat.java

```diff
@@ -1,9 +1,9 @@
 package com.linkedin.venice.hadoop.input.kafka;
```

Comment:
If the goal is to axe `hadoop` from package names as much as possible (is that the goal? IDK if I understand the goal of this PR, actually; but anyway, if that is the goal...) then why leave the test classes in the `hadoop` package?

The convention is that a given test class should be in the same package as the main code it intends to exercise. This enables us to limit visibility rules to package-private (i.e. no visibility modifier at all), so that APIs which need to do special dependency injection are less visible.

The `KafkaInputFormat` class has a visibility change due to this mismatch.

Reply:
The goal is to move non-Hadoop parts out of the Hadoop package. Nisarg pointed out that this input format is tied to MR framework and by that token, Hadoop, so it should stay within that package.

I think at this point, the main class for KafkaInputFormat resides in the same pacakge as the tests for it, both unit and integration.

Reply:
Got it. So then `KafkaInputFormat:: getLatestOffsets` can be made package-private, rather than public.

Reply:
fixed.